Source/CarthageKit/Errors.swift

```diff
@@ -71,6 +72,10 @@ public enum CarthageError {
 	/// them.
 	case NoSharedSchemes(ProjectLocator)
 
+	/// The private cartfile contains dependencies that are already present in
+	/// the main cartfile.
+	case DuplicateDependencies([(ProjectIdentifier, /*label*/ String)])
```

Comment:
We could also implement `Printable` that way, making the string formatting below easier.